addons/web/static/src/legacy/js/public/public_widget.js

```diff
@@ -19,18 +17,12 @@ import Widget from 'web.Widget';
  * @todo Merge with 'PublicWidget' ?
  */
 var RootWidget = Widget.extend({
-    custom_events: _.extend({}, Widget.prototype.custom_events || {}, {
-        'registry_update': '_onRegistryUpdate',
-        'get_session': '_onGetSession',
-    }),
```

Comment:
Removing this but keeping the `_onRegistryUpdate`, same for `_onGetSession`... I don't follow this commit :grimacing: And what about the code using the `get_session` event? :thinking:

Reply:
Whoops, need to remove those dead methods. `get_session` event is now handled by `ComponentAdapter` that I've set as this widget's parent, it does a bunch of `trigger_up` and `call_service` translation.